app/controllers/application_controller/authentication_dependency.rb

```diff
@@ -48,6 +51,11 @@ def logged_in?
     !current_user.nil?
   end
 
+  def log_out
```

Comment:
We might want to make use of the flash to tell the user that they were logged out of Classroom.